cfpb/design-system
# Hide content when the expandable is collapsed

Adds `display: none` to the expandable's content when it is collapsed.

## Changes

- Adds placeholder links to the expandable content so they contain focusable content.
- Adds event handlers for `expandBegin` and `collapseEnd` events, which add and remove the `u-hidden` class to the expandable content DOM node.

## Testing

1. Pull branch, run `gulp build`
2. `cd docs && yarn && yarn start`
3. Visit http://localhost:4000/design-system/components/expandables
4. Using the tab key, pay attention to what has focus. When collapsed, focus should jump from the expandable header to the next focusable element on the page. When expanded, focus should jump from the expandable header, to the link in the expandable content, to the next focusable element on the page.

## Screenshots

<img width="842" alt="Screen Shot 2020-01-21 at 5 12 33 PM" src="https://user-images.githubusercontent.com/704760/72847994-9fe86500-3c71-11ea-8b8a-714db6921a50.png">
